ui/lib/apps/TopSQL/utils/specialRecord.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright 2021 PingCAP, Inc. Licensed under Apache-2.0.
+
+import type { PlanRecord } from '../pages/List/ListDetail/ListDetailTable'
+import type { SQLRecord } from '../pages/List/ListTable'
+
+const OVERALL_IDENTIFIER = '__OVERALL_IDENTIFIER__'
+
+export const isOverallRecord = (r: PlanRecord) => {
+  return r.plan_digest === OVERALL_IDENTIFIER
+}
+
+export const createOverallRecord = (records: PlanRecord[]) => {
```

Comment:
I think this should be handled by ngm @zhongzc 
The front-end should not have any logic about the computation.

Reply:
Oh, yes, there is wrong computation here.

Reply:
> I think this should be handled by ngm

https://github.com/pingcap/ng-monitoring/pull/93